qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml

```diff
@@ -70,7 +70,10 @@ QtObject {
       property real height: 48 * QgsQuick.Utils.dp
       property real cornerRadius: 0 * QgsQuick.Utils.dp
       property int fontPixelSize: 48 * QgsQuick.Utils.dp
-
+      property real sideMargin: 12 * QgsQuick.Utils.dp
+      property real outerlMargin: 20 * QgsQuick.Utils.dp
+      property int fontPointSize: 16
+      property int labelPointSize: 14
```

Comment:
Can we add brief comments to these properties? I am getting lost especially what exactly is the sideMargin and outerMargin...